modules/ROOT/pages/index.adoc

```diff
@@ -118,13 +138,59 @@ image::create-database-access.png[Creating database access]
 
 . To set up a secure connection, you will need to retrieve the root certificate for your Capella instance.
 +
-[lowerroman]
 .. Click on btn:[Security Certificate] in the left menu.
 .. Click on btn:[Download] to transfer a copy of the root certificate to your local machine.
 +
 image::download-root-certificate.png[]
 --
 
+Capella Columnar::
++
+--
+
+For details on creating a Capella Columnar instance,
+first see xref:columnar:admin:prepare-project.adoc[Creating a Capella Columnar Cluster].
+
+[#capella-columnar-connection-string]#You will need to access the Capella administration console to get the connection string for your columnar database.#
+
+. Sign in to your Capella instance as an Administrator.
```

Comment:
Sign in not as an Administrator but as Organization owner or Project owner https://preview.docs-test.couchbase.com/columnar-ga/columnar/admin/auth/auth-data.html#prerequisites

Reply:
Fixed.